docs/docs/ecommerce-tutorial/index.md

```diff
@@ -0,0 +1,551 @@
+* Write your own simple server and deploy it somewhere, making it accessible via an endpoint
+* Write and deploy a hosted serverless function through a service like AWS Lambda or Google Cloud
+
+For this tutorial, you’ll use a serverless function (through AWS Lambda, though the steps in this tutorial will allow you to deploy to other cloud services in almost the same way). You also benefit from not having to keep a server running, because you only pay for the server time you use. Many companies that provide the ability to deploy serverless functions have a free tier that covers anywhere from several thousand to a million requests per month.
+
+[Insert a diagram showing a request flowing from Your static Gatsby site ===> Your serverless function ===> Stripe's servers and back again]
+
+
+# Setting up a Gatsby site
+
+Create a new Gatsby project by running the `gatsby new` command in the terminal and change directories into the new project you just started:
+
+```shell
+gatsby new ecommerce-gatsby-tutorial https://github.com/gatsbyjs/gatsby-starter-default
+cd [name of project]
+```
+
+
+## Extending Gatsby through an e-commerce plugin
+
+You can extend the functionality of this default starter with plugins. One such plugin is `gatsby-plugin-stripe-checkout`, which you’ll install in this project:
+
+```shell
+npm install gatsby-plugin-stripe-checkout
+```
+
+Open the root site directory in a text editor and navigate to `gatsby-config.js` and add stripe-checkout plugin to `gatsby-config.js` in the plugins section. Your `gatsby-config.js` should look like the following code example:
+
+```jsx
+module.exports = {
+ siteMetadata: {
+   title: 'Gatsby Default Starter',
+ },
+ plugins: ['gatsby-plugin-react-helmet', 'gatsby-plugin-stripe-checkout'],
+}
+```
+
+
+## See your site hot reload in the browser!
+
+Run `gatsby develop` in the terminal, which starts a development server and reloads changes you make to your site so you can preview them in the browser (you can turn off the server by pressing Ctrl + C). Open up your browser to localhost:8000 and you should see a default homepage.
+
+> **NOTE**: If you have already started your gatsby development server using `gatsby develop` you will need to restart the server by pressing CTRL + C and running the command again to see changes in your gatsby.config reflected on localhost***
+
+
+## How does the plugin work?
+
+Stripe Checkout processes payments with information we send it, you can read more about how it works in Stripe’s docs. The Gatsby plugin, `gatsby-plugin-stripe-checkout`, will add this snippet:
+
+```shell
+<script src="https://checkout.stripe.com/checkout.js"></script>
+```
+
+to the end of the `<body>` tag across all of your pages in your Gatsby project, allowing you to call the Stripe checkout methods from Stripe’s API to create charges.
+
+If you’d like more extensive access to Stripe data, explore more Gatsby / Stripe plugins.
```

Comment:
What about something like this?

_If you want to further customise the checkout process or pull Stripe data into your site, check out [Gatsby's plugin library for more Stripe plugins](https://www.gatsbyjs.org/plugins/?=stripe) ._

And then delete the note below about the Stripe Checkout Elements plugin.